### PR TITLE
Windows touch fix 

### PR DESCRIPTION
**Pull request by daschuer**
This adds some debug messages to touch events for digging into bug https://bugs.launchpad.net/mixxx/+bug/1662794.

### CODE REVIEW

**Comment by daschuer** [comment]
Finally this works nicely, on Win10.

File: src/mixxxapplication.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include "library/crate/crateid.h"
 #include "control/controlproxy.h"
 #include "mixxx.h"
+#include "util/synthesizedmousewvents.h"
```
Comment (rryan) [inline src/mixxxapplication.cpp]: typo in filename (wvents -> events)

**Comment by rryan** [comment]
LGTM, thanks

**Comment by daschuer** [comment]
Thank you for review.

**Pull request merged**